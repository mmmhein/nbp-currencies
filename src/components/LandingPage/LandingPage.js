import React from 'react';

import FollowedRateList from '../FollowedRateList';
import InfoBanner from '../InfoBanner';
import Content from '../Content';

function LandingPage() {
  return (
    <article>
      <InfoBanner>
        <strong data-testid="info-banner-favorite">Ulubione waluty</strong>
      </InfoBanner>
      <Content>
        <FollowedRateList />
      </Content>
    </article>
  );
}

export default LandingPage;
