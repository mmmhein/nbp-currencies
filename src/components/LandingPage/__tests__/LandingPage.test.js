import React from 'react';
import { create } from 'react-test-renderer';

import LandingPage from '../LandingPage';

jest.mock('../../FollowedRateList', () => 'FollowedRateList');
jest.mock('../../InfoBanner', () => 'InfoBanner');

describe('COMPONENT - LandingPage', () => {
  it('renders correctly', () => {
    const component = create(<LandingPage />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
