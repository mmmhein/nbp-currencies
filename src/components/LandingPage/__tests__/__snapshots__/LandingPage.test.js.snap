// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - LandingPage renders correctly 1`] = `
.c0 {
  max-width: 100rem;
  padding: 0 2rem;
  margin: 0 auto;
}

<article>
  <InfoBanner>
    <strong
      data-testid="info-banner-favorite"
    >
      Ulubione waluty
    </strong>
  </InfoBanner>
  <section
    className="c0"
  >
    <FollowedRateList />
  </section>
</article>
`;
